internal/jobs_test.go

```diff
@@ -89,13 +89,13 @@ func TestAccJobsApiFullIntegration(t *testing.T) {
 	require.NoError(t, err)
 
 	cancelledRun, err := w.Jobs.CancelRunAndWait(ctx, jobs.CancelRun{
-		RunId: runNowResponse.RunId,
+		RunId: runNowResponse.Response.RunId,
```

Comment:
Unlike for Python, we have the opportunity here to return 3 values: the response, the waiter, the error.

Then there's no need for calls like this.

Reply:
added direct fields for polling continuation across process boundary